src/Util/FileUploader.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace App\Util;
+
+use Gedmo\Sluggable\Util\Urlizer;
+use phpDocumentor\Reflection\DocBlock\Tags\Var_;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\HttpFoundation\Request;
+
+class FileUploader
+{
+    /**
+     * The directory where uploaded files will be placed.
+     *
+     * @var string
+     */
+    private $uploadDirectory;
+
+    /**
+     * The directory where file chunks will be stored.
+     *
+     * @var string
+     */
+    private $chunksDirectory;
+
+    public function __construct(string $homedirPrefix)
+    {
+        $this->uploadDirectory = $homedirPrefix . DIRECTORY_SEPARATOR . 'upload' . DIRECTORY_SEPARATOR . 'files';
+        $this->chunksDirectory = $homedirPrefix . DIRECTORY_SEPARATOR . 'upload' . DIRECTORY_SEPARATOR . 'chunks';
+    }
+
+    /**
+     * File uploader service.
+     *
+     * @param Request $request Symfony request instance.
+     *
+     * @return array
+     */
+    public function upload(Request $request): array
+    {
+        dump($request);
+        $totalChunks = $request->get('dztotalchunkcount');
+        $chunkIndex = $request->get('dzchunkindex');
+        /** @var UploadedFile $uploadedFile */
+        $uploadedFile = $request->files->get('file');
```

Comment:
Can also use filebag?

Reply:
I'll try it, should work i guess

Reply:
Not sure, if there is getFileBag function. I think this is the right way to get it

Reply:
filebag is in the request, but it's an option to use.